README.md

```diff
@@ -1,135 +1,268 @@
+includes a small set of distributions and corresponding file corpora,
+for example PDF. The PDF distribution can be launched from the root
+folder with the following command: `workbench.py pdf test_files/pdf`.
+
+The workbench script builds a docker image for the distribution on the
+fly and launches an instance of the FAW. The FAW user interface is
+accessible via a webbrowser at `http://localhost:8123` (by default).
+
+By default, FAW is launched in [developer mode](#developer-friendly-features).
+
+To create your own distribution, take a look at
+[this guide](./docs/tutorial-distribution.md).
+
+TODO: Describe teaming mode
+
+# Capabilities of the FAW
+
+The FAW offers a number of features/capabilities to the user out of the
+box that we describe in the following sections.
+
+## Filtering and Discrepancy Analysis
+
+FAW distributions typically include file parsers for parsing files and
+extracting features. While manual validation of features is possible, it
+is more convenient to have automatic decisions based on user-defined
+criteria. The FAW offers a simple, custom domain-specific language (DSL)
+for expressing decision criteria based on *filters* and *outputs*.
+Filters are groups of expressions and are matched against parser
+generated features while outputs, expressed in terms of compound boolean
+expressions over filters, represent aggregations of filter criteria.
+
+TODO: Point to a file as example
+
+The decision DSL adjustments impact the FAW UI, with filter and output
+clauses represented as radio buttons in the UI. Selecting these buttons
+filters the file list to display only the files that meet the criteria
+specified in the DSL.
+
+The FAW also enables users to upload structured JSON files containing
+reference decisions for files in a corpus. These "gold standard"
+decisions are automatically compared to the decisions made by the
+current DSL, highlighting any discrepancies for users to investigate and
+identify files that do not match expectations.
+
+## Extensibility
+
+The FAW is designed to accommodate diverse input formats, parsers, and
+visualization tools. It employs a plugin architecture with extension
+points and an API, enabling users to create custom plugins tailored to
+their specific corpus and workflow. Plugins can be developed for parsing
+new file formats and visualizing data from the corpus. The FAW supports
+various plugin types, which are further described in subsequent
+sections.
+
+### Parser Plugins
+
+The FAW supports parser plugins that validate file formats and extract
+format-specific features. These plugins execute external commands to
+parse files, capturing the exit code and standard output/error. The FAW
+allows mapping of these outputs to human-understandable features using
+regular expressions, with options for replacement and capture. The
+extracted features are stored in the database and can contribute to the
+valid/invalid decisions via the decision DSL mentioned in the previous
```

Comment:
Minor note: Could link to said previous section/subsection here

Reply:
A link has been added as of commit [823826e](https://github.com/GaloisInc/FAW/commit/823826e0f175918ccbaf9ff75a41d82a4b29c767).